examples/basics/plotting/grid_two_row_two_col_.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+# vispy: gallery 30
+# -----------------------------------------------------------------------------
+# Copyright (c) 2015, Vispy Development Team. All Rights Reserved.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+# -----------------------------------------------------------------------------
+"""
++-----+-----------+
+| 1   |     2     |
++-----+----+------+
+|   3      |   4  |
++----------+------+
```

Comment:
Wait, question: If the column width does not change, then is the entire column "locked" to the same width? Didn't we agree on flexible widths without collapse?